server.py

```diff
@@ -631,6 +634,21 @@ def get_prompts_scaffold():
         },
     ]
 
+# Submit all reports to AVID
+# Logs the responses
+def submit_reports_to_AVID(reports):
+    #Set up the connection to AVID
+    root = environ.get('AVID_API_URL')
+    api_key = environ.get('AVID_API_KEY')
+    key = {"Authorization": api_key}
+
+    for r in reports:
+        new_report = utils.convert_indie_label_json_to_avid_json(r)
+        url = root + "submit"
+        response = requests.post(url, json=json.loads(new_report), headers=key) # The loads ensures type compliance
+        uuid = response.json()
+        print("AVID API response:", response, uuid)
```

Comment:
minor: just a note that we probably want to remove this for the final version (or put it within a debug flag)! Note to myself that I should take a pass at the old code to remove remaining print statements that aren't under a debug flag.